client/src/components/utils/Footer/style.css

```diff
@@ -0,0 +1,96 @@
+.footer {
+  background: var(--main-color--light);
+  color: var(--main-white);
+  display: flex;
+  flex-direction: column;
+}
+
+.container {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+.footer h2 {
+  color: var(--main-white);
+  font-weight: 700;
+  font-size: 1.75rem;
+}
+
+.footer__section {
+  width: 30%;
```

Comment:
This percentage takes 90% of the container div, also the container div takes 95% from the parent container, so the actual margin of the footer sections is 5% + 10% of the div container,
Could you please check your margins percentage again?